tasks/task05/task05.py

```diff
@@ -0,0 +1,34 @@
+def create_user(name, surname, **kwargs) -> dict:
```

Comment:
for age it would be nice to use a named (keyword) parameter with a default value here